docs/_pages/collections.md

```diff
@@ -40,7 +40,10 @@ collection.Should().StartWith(new[] { 1, 2 });
 collection.Should().EndWith(8);
 collection.Should().EndWith(new[] { 5, 8 });
 
-collection.Should().BeSubsetOf(new[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, });
+collection.Should().BeSubsetOf(new[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, }); // It should be contained or equivalent to the supetset.
+collection.Should().BeProperSubsetOf(new[] { 1, 2, 5, 6, 7, 8 }); // It should be contained but not equivalent to the superset.
+collection.Should().BeSupersetOf(new[] { 1, 2, 5, 8 }); // It should contain or be equivalent to the subset.
+collection.Should().BeProperSupersetOf(new[] { 1, 5, 2 }); // It should contain but not be equivalent to the subset.
```

Comment:
🔧This won't fit on a single. See https://fluentassertions.com/collections/. Using comments on top of the examples, or separate section will probably work better. Also note that you can run the documentation website locally from the `docs` folder.